src/metabase/db/custom_migrations.clj

```diff
@@ -1275,6 +1275,6 @@
   (set-jdbc-backend-properties!)
   (let [scheduler (qs/initialize)]
     (qs/start scheduler)
-    (qs/delete-trigger scheduler (triggers/key "metabase.task.send-pulses.job"))
```

Comment:
urgh, it was swapped.

the fact that we have to do this every time we remove a job class is also bad. I'll explore a way so this is done automatically.

Reply:
triggers get deleted with the job, as you said below. So we don't need line 1266